Raven.Database/Bundles/Replication/Triggers/VirtualAttachmentDeleteTrigger.cs

```diff
@@ -52,12 +52,15 @@ public override void OnDelete(string key)
 
         public override void AfterDelete(string key)
         {
+            var now = DateTime.UtcNow;
             var metadata = new RavenJObject
             {
                 {Constants.RavenDeleteMarker, true},
                 {Constants.RavenReplicationHistory, deletedHistory.Value},
                 {Constants.RavenReplicationSource, Database.TransactionalStorage.Id.ToString()},
-                {Constants.RavenReplicationVersion, ReplicationHiLo.NextId(Database)}
+                {Constants.RavenReplicationVersion, ReplicationHiLo.NextId(Database)},
+                {Constants.LastModified,now},
+                {Constants.RavenLastModified,now }
```

Comment:
@talweiss1982 Do we need both, LastModified and RavenLastModified set in metadata?